projects/pgai/tests/vectorizer/conftest.py

```diff
@@ -70,3 +71,9 @@ def postgres_container():
         driver=None,
     ) as postgres:
         yield postgres
+
+@pytest.fixture(scope="session")
+def ollama_container():
+    with OllamaContainer(image="ollama/ollama:latest") as ollama:
```

Comment:
Could this use a volume mount to cache models?

Reply:
Done. 

```python
# Passing the ollama_home lets us reuse models that have already
# been pulled to the `~/.ollama` path on the host machine.
ollama_home=Path.home() / ".ollama",
```